### PR TITLE
App Layout usage and specs pages added to the site

### LINKED ISSUE #1471: Application Layout: Usage & Specifications

### PR DESCRIPTION
**Pull request by raquelarrojo**
**Checklist**
_(Check off all the items before submitting)_
- [x] Build process is done without errors and all tests pass in `/lib` directory.
- [x] Self-reviewed the code prior to submitting.
- [x] Meets accessibility standards.
- [x] Added/updated documentation to `/website` as needed.
- [x] Added/updated tests as needed.

**Purpose**
Add App Layout usage and specs page to the site.

**Description**
Added App Layout usage and specs page to the site.

**Closes #1471**

### CODE REVIEW

File: website/screens/components/application-layout/usage/ApplicationLayoutUsagePage.tsx [hunk 1]
```diff
@@ -2,77 +2,129 @@ import {
...
-        </Figure>
         <DxcParagraph>
-          Overlay and push interactions by the sidenav component can alter the
-          way the layout is displayed. For overlay, the sidenav simply slides
-          over the content with no changes to position. The push interaction
-          moves the main container to the side depending on the width of the
-          sidenav.
+          Breakpoints define resolutions at which screen components adjust to
+          offer an optimal user experience across screen sizes and devices.
+          We&#39;ve defined five different breakpoints to accommodate multiple
```
Comment (Lisa159) [inline website/screens/components/application-layout/usage/ApplicationLayoutUsagePage.tsx:71]: Maybe change the code `&#39;` with a normal `'`.

File: website/screens/components/application-layout/usage/ApplicationLayoutUsagePage.tsx [hunk 2]
```diff
@@ -2,77 +2,129 @@ import {
...
+              <td>
+                <Code>xlarge</Code>
+              </td>
+              <td>1440</td>
+              <td>90</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+        <DxcParagraph>
+          The image below illustrates how we&#39;ve used data on the most
```
Comment (Lisa159) [inline website/screens/components/application-layout/usage/ApplicationLayoutUsagePage.tsx:121]: Maybe change the code `&#39;` with a normal `'`.

File: website/screens/components/application-layout/usage/ApplicationLayoutUsagePage.tsx (hunk 1 above)
Comment (raquelarrojo) [reply website/screens/components/application-layout/usage/ApplicationLayoutUsagePage.tsx:71]: `&#39;` is the same as `'`, and we avoid the error that gives in Visual Studio.

File: website/screens/components/application-layout/usage/ApplicationLayoutUsagePage.tsx (hunk 2 above)
Comment (raquelarrojo) [reply website/screens/components/application-layout/usage/ApplicationLayoutUsagePage.tsx:121]: `&#39;` is the same as `'`, and we avoid the error that gives in Visual Studio.

**Review by Lisa159 — APPROVED** [review]

**Pull request merged**